index.html

```diff
@@ -254,7 +254,7 @@ <h2>Tonic</h2>
                             </div>
                         </div>
                         <div class="card-button">
-                            <button class="button-work-project">SEE PROJECT</button>
+                            <button onclick="getProject(3)" class="button-work-project">SEE PROJECT</button>
```

Comment:
- [ ] Great job working on the functionality for the pop up and design but I think you forget to remove the projects section and make it to load dynamically through the js 😊🚀🔥

![image](https://user-images.githubusercontent.com/58553711/202313834-86b945a4-d540-4c4d-8cf4-6f36afd56da4.png)